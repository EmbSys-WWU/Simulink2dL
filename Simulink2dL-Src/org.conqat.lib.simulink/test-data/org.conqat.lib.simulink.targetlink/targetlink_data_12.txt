{'logdata',{'plotchannels',[]},'output',{'variable','f_taskTime'},'date',734875.60776906251}