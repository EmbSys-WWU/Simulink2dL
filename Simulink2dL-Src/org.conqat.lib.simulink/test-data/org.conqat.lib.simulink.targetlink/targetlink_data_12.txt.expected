struct (
  'logdata' -> struct (
    'plotchannels' -> '[]'
  )
  'output' -> struct (
    'variable' -> 'f_taskTime'
  )
  'date' -> '734875.60776906251'
)